apps/veil/src/pages/tournament/ui/delegator-table/delegator-rewards.tsx

```diff
@@ -21,39 +51,134 @@ export const DelegatorRewards = () => {
             Total Rewards Earned
           </Text>
           <Text small color='text.secondary'>
-            Cumulative from all epochs, voting and LPs rewards
+            Cumulative from all epochs, and voting rewards
```

Comment:
question: shouldn't it be "Cumulative voting rewards (in UM) from all epochs" like in the design?

Reply:
Yeah, absolutely!